src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReflectionVirtualInvokeMapNode.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Collections.Generic;
+
+using Internal.Text;
+using Internal.TypeSystem;
+using Internal.NativeFormat;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    /// <summary>
+    /// Represents a map containing the necessary information needed to resolve 
+    /// a virtual method target called through reflection.
+    /// </summary>
+    internal sealed class ReflectionVirtualInvokeMapNode : ObjectNode, ISymbolNode
+    {
+        private const uint GenericVirtualMethod = 1;
```

Comment:
Reusing VirtualInvokeTableEntry.GenericVirtualMethod from MappingTableFlags.cs instead.

---
In reply to: [108004794](https://github.com/dotnet/corert/pull/3100#discussion_r108004794) [](ancestors = 108004794)